pocs/utils/theskyx.py

```diff
@@ -1,18 +1,19 @@
 import socket
 
-from pocs.base import PanBase
 from pocs.utils import error
+from pocs.utils.logger import get_root_logger
 
 
-class TheSkyX(PanBase):
+class TheSkyX(object):
 
     """ A socket connection for communicating with TheSkyX
 
 
     """
 
     def __init__(self, host='localhost', port=3040, connect=True, *args, **kwargs):
-        super(TheSkyX, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
Just curious, why still have an explicit call to `super().__init__()` when subclassing from `object`? Does this actually do anything, or is it just there in case the base class is changed?

Reply:
No, I was going to remove it but kept it for good measure. Will remove.